diff --git a/src/sound/jack/sound.h b/src/sound/jack/sound.h
index d4f5de2..59c9851 100644
--- a/src/sound/jack/sound.h
+++ b/src/sound/jack/sound.h
@@ -91,7 +91,7 @@ public:
         bJackWasShutDown ( false ),
         fInOutLatencyMs ( 0.0f )
     {
-        QString strJackName = QString ( APP_NAME );
+        QString strJackName = qEnvironmentVariable( "JAMULUS_JACK_NAME", APP_NAME );
 
         if ( !strJackClientName.isEmpty() )
         {
